src/PureCloud.Client/Apis/PresenceApi.cs

```diff
@@ -22,6 +22,49 @@ public PresenceApi(IHttpClientFactory httpClientFactory, IOptions<PureCloudJsonS
         _options = options.Value.JsonSerializerOptions;
     }
 
+    /// <inheritdoc />
+    public async Task<bool> DeletePresenceDefinitionAsync(string definitionId, CancellationToken cancellationToken = default)
+    {
+        ArgumentException.ThrowIfNullOrEmpty(definitionId);
+
+        var uri = UriHelper.GetUri($"api/v2/presence/definitions/{Uri.EscapeDataString(definitionId)}", null);
+
+        var response = await _httpClient.DeleteAsync(uri, cancellationToken);
+
+        response.EnsureSuccessStatusCode();
```

Comment:
Don't use EnaureSucessStatusCode instead return result.IsSuccessStatusCode. Apply the same on all methods that return bool